lua/nvim-tree.lua

```diff
@@ -112,6 +112,8 @@ function M.on_keypress(mode)
   else
     lib.open_file(mode, node.absolute_path)
   end
+
+  renderer.draw(lib.Tree, true)
```

Comment:
is this required ? if it wasn't there i guess it was for a reason :)

Reply:
This is what makes file highlighting work when opening a file from nvim-tree. Otherwise, the highlighting doesn't take place right after editing a file (it only happens with files specifically though).

Reply:
i see, maybe placing this in the open_file call would be more appropriate then.

Reply:
It makes sense. I moved it, just not sure I placed it into the right place in `open_file`